vaccine_feed_ingest/runners/il/sfsites/normalize.py

```diff
@@ -0,0 +1,105 @@
+#!/usr/bin/env python
+
+import datetime
+import json
+import pathlib
+import sys
+
+from vaccine_feed_ingest.utils.normalize import organization_from_name
+
+# import schema
+site_dir = pathlib.Path(__file__).parent
+state_dir = site_dir.parent
+runner_dir = state_dir.parent
+root_dir = runner_dir.parent
+sys.path.append(str(root_dir))
+from schema import schema  # noqa: E402
+
+
+def get_statewide_ids(statewide_filepath: str) -> set:
+    statewide_ids = set()
+
+    with statewide_filepath.open() as file:
+        for site_json in file:
+            parsed_site = json.loads(site_json)
+
+            statewide_ids.add(parsed_site["Id"])
+
+    return statewide_ids
+
+
+def normalize(site: dict, statewide_ids: set, timestamp: str) -> dict:
+    location_id = site["Id"]
+    name = site["Testing_Center__c"]
+    notes = []
+
+    if "Location_Type__c" in site:
+        notes.append(site["Location_Type__c"])
+
+    if location_id not in statewide_ids:
+        notes.append("Not open to all Illinois residents")
```

Comment:
@yotann @edcous Added this bit here to the `notes` field, lmk what you think.

Maybe the copy could be better? Thoughts?

Reply:
The issue is that a lot of the locations don't care where in Illinois you live are on the big list, but not the small one. As an example, if I pick the 2nd location (Advocate Aurora South Suburban Hospital) it doesn't ask about residency at all. If a note was added saying it wasn't open to all residents that would be wrong, yet it is not on the statewide list.

Reply:
@edcous Yeah, I just called them too, and it sounds like they're really just available to the general public. I'll back this change out so it's not confusing to users.